docs/user/general-api.md

```diff
@@ -61,13 +62,14 @@ class SampleApplication : Application() {
             applicationContext,
             // Here, `settings()` is a method to get user preferences, specific to
             // your application and not part of the Glean SDK API.
-            uploadEnabled = settings().isTelemetryEnabled
+            uploadEnabled = settings().isTelemetryEnabled,
+            buildInfo = GleanBuildInfo.buildInfo
```

Comment:
I wonder if future versions of the language bindings could even do without this, with Glean automatically fetching the generated code.

Reply:
Not really? We need to generate this in the final application because only there the build info is valid (or even existent?)
The only way we get to that data is if someone gives it to us (this is the same problem as with `registerPings`: We don't know about them unless they get passed in)